presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestPrestoDriverUri.java

```diff
@@ -176,6 +178,24 @@ public void testUriWithoutCompression()
         assertEquals(parameters.getProperties().getProperty(DISABLE_COMPRESSION.getKey()), "true");
     }
 
+    @Test
+    public void testUriWithoutHttp2()
+            throws SQLException
+    {
+        PrestoDriverUri parameters = createDriverUri("presto://localhost:8080/blackhole?disableHttp2=true");
```

Comment:
Curious who uses this HTTP flag?  Does this have some sort of significance to the HTTP client?

Reply:
We have a few customers who are behind an LB that accepts HTTP2 which `okhttp3` defaults to in recent Java 8 and 11 versions. The DB doesn't accept HTTP2 when it's forwarded along - these are the folks that use the flag.

I was thinking, let's make it a flag just in-case someone has an existing config that needs HTTP2 and have the option to disable it for folks who need it in their setups.